### PR TITLE
String format fix

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: src/contract.js [hunk 1]
```diff
@@ -4,11 +4,15 @@ const _ = require('lodash');
 // Internal Imports
 const Formatter = require('./formatter');
 const Utils = require('./utils.js');
+const Encoder = require('./encoder');
 
 const SEND_AMOUNT = 0;
 const SEND_GASLIMIT = 250000;
 const SEND_GASPRICE = 0.0000004;
 
+const MAX_BYTES_PER_ARRAY_SLOT = 64;
```
Comment (frankobe) [inline src/contract.js]: this is bodhi-specific constants, leave it as it is now, should move it to `bodhi-core.js` later

File: src/contract.js [hunk 2]
```diff
@@ -78,27 +82,36 @@ class Contract {
...
-          hex = Utils.addressToHex(args[index]);
+        case 'address': {
+          hex = Encoder.addressToHex(args[index]);
           dataHex = dataHex.concat(hex);
           break;
-        case 'bytes32[10]':
-          hex = Utils.stringArrayToHex(args[index], 10);
-          dataHex = dataHex.concat(hex);
+        } 
+        case 'bytes32[10]': {
```
Comment (frankobe) [inline src/contract.js]: it should be able to handle arbitrary length of byte32 array. Leave it as it is for now, should add `//TODO` comment

File: src/utils.js [hunk 3]
```diff
@@ -728,12 +590,6 @@ module.exports = {
...
   trimHexPrefix: trimHexPrefix,
-  getFunctionHash: getFunctionHash,
-  addressToHex: addressToHex,
-  stringToHex: stringToHex,
-  stringArrayToHex: stringArrayToHex,
-  uint8ToHex: uint8ToHex,
-  uint256ToHex: uint256ToHex,
   hexToNumber: hexToNumber,
   chunkString: chunkString,
 };
```
Comment (frankobe) [inline src/utils.js]: new line EOF

File: src/contract.js (hunk 1 above)
Comment (dwalintukan) [reply src/contract.js]: Ya I will make this more generic later. But actually 64 is always the padded length for any qtum-cli param. Think the naming is a little off though.

File: src/contract.js (hunk 2 above)
Comment (dwalintukan) [reply src/contract.js]: Plan to make it more variable later. Just getting it working for now.

File: src/utils.js (hunk 3 above)
Comment (dwalintukan) [reply src/utils.js]: done

**Pull request merged**